qiime_studio/server.py

```diff
@@ -25,6 +33,7 @@ def hello():
 
 def start_server():
     studio.debug = True
+    studio.before_request(flush_stdio)
```

Comment:
`after_request` might be better. I noticed when I crashed everything (forgot to stash my changes to `qiime2`) that I didn't see the stack trace in the terminal until the next request.